### ISSUE TITLE
iTerm setting does not seem to work

### ISSUE THREAD

**Issue by npostman** [adamwathan/sublime-phpunit#12]
Hi Adam,
I'm working on your TDD course and am trying to get your workflow set up. So far so good, but when I add the ""phpunit-sublime-terminal": "iTerm"" setting to my Sublime settings nothing happens when running the commands. Without the setting it works like charm with the build in terminal.

Any hints on where to start my trouble shoot investigation?

**Comment by LJ1001** [comment]
I have the same issue. I can also verify that the commands are triggered. (`sublime.log_commands(True)`) This is why I'm thinking the problem has nothing to do with this package but with sublime itself.

But I have also no solution right now.

**Comment by rankogit** [comment]
It is working for me. I make this comment only because as it wasn't working for me I found this post and was tempted to stop trying to work with iTerm. My issue was that the package worked with Terminal. Then I closed out my test and altered the settings file as is necessary to get iTerm to work. I ran the "run all tests" and it didn't work. My new understanding is at least one test (like ExampleTest.php in laravel) must me open to run the tests. Long story short, the iTerm issue may be a problem for some but for me at least it wasn't understanding how the package worked.
